.circleci/config.yml

```diff
@@ -20,37 +29,61 @@ executors:
       - image: cimg/node:<< parameters.tag >>
 
 commands:
-  install-dependencies:
+  cmd_install-dependencies:
```

Comment:
Adding these prefixes to make it easier to know if you invoking a command vs pipeline vs workflow